bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml

```diff
@@ -229,7 +230,9 @@ spec:
           - create
         serviceAccountName: aws-load-balancer-operator-controller-manager
       deployments:
-      - name: aws-load-balancer-operator-controller-manager
+      - label:
+          control-plane: controller-manager
```

Comment:
This label is new in CSV but it [was there](https://github.com/openshift/aws-load-balancer-operator/blob/1a2c320b27d4b5bb1a9c259db0796c462a11cc98/config/manager/manager.yaml#L13-L14) in `manager.yaml` since the beginning so I suppose it's just a fixed bug in the latest operator-sdk.